src/rendering/RenderUtil.cc

```diff
@@ -1064,9 +1070,7 @@ void RenderUtil::Update()
         // update actor trajectory animation
         math::Pose3d globalPose;
         if (entityPoses.find(it.first) != entityPoses.end())
-        {
           globalPose = entityPoses[it.first];
-        }
```

Comment:
Had to remove some "{ }" because the RenderUtil::Update function produce a codecheck error as it exceeded 500 non-comment line

Reply:
Ok great